google/cloud/aiplatform/training_jobs.py

```diff
@@ -3346,3 +3347,255 @@ def _run(
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ):
+        """Constructs a AutoML Text Training Job.
+
+        Args:
+            display_name (str):
+                Required. The user-defined name of this TrainingPipeline.
+            prediction_type (str):
+                The type of prediction the Model is to produce, one of:
+                    "classification" - A classification model analyzes text data and
+                        returns a list of categories that apply to the text found in the data.
+                        AI Platform offers both single-label and multi-label text classification models.
+                    "extraction" - An entity extraction model inspects text data
+                        for known entities referenced in the data and
+                        labels those entities in the text.
+                    "sentiment" - A sentiment analysis model inspects text data and identifies the
+                        prevailing emotional opinion within it, especially to determine a writer's attitude
+                        as positive, negative, or neutral.
+            multi_label (bool):
+                Required and only applicable for text classification task. If false, a single-label (multi-class) Model will be trained (i.e.
+                assuming that for each text snippet just up to one annotation may be
+                applicable). If true, a multi-label Model will be trained (i.e.
+                assuming that for each text snippet multiple annotations may be
+                applicable).
+            sentiment_max (int):
+                Required and only applicable for sentiment task. A sentiment is expressed as an integer
+                ordinal, where higher value means a more
+                positive sentiment. The range of sentiments that
+                will be used is between 0 and sentimentMax
+                (inclusive on both ends), and all the values in
+                the range must be represented in the dataset
+                before a model can be created.
+                Only the Annotations with this sentimentMax will
+                be used for training. sentimentMax value must be
+                between 1 and 10 (inclusive).
+            project (str):
+                Optional. Project to run training in. Overrides project set in aiplatform.init.
+            location (str):
+                Optional. Location to run training in. Overrides location set in aiplatform.init.
+            credentials (auth_credentials.Credentials):
+                Optional. Custom credentials to use to run call training service. Overrides
+                credentials set in aiplatform.init.
+        """
+        super().__init__(
+            display_name=display_name,
+            project=project,
+            location=location,
+            credentials=credentials,
+        )
+        self._prediction_type = prediction_type
```

Comment:
These attributes don't seem necessary since they are enveloped in training_task_input_dict.

Reply:
Good catch. Missed these after refactoring.

Reply:
Done